Actions/AL-Go-Helper.ps1

```diff
@@ -1275,13 +1278,16 @@ function CloneIntoNewFolder {
     invoke-git clone $serverUrl
 
     Set-Location *
-    invoke-git checkout $ENV:GITHUB_REF_NAME
+    invoke-git checkout $updateBranch
 
-    if ($branch) {
+    $branch = ''
+    if (!$directCommit) {
```

Comment:
```suggestion
    if ($newBranchPrefix) {
```
Then you can remove the directCommit parameter

Reply:
NewBranchPrefix is always set - it is the prefix IF directCommit is set.